CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Integrate `index_origin` with all the library ([1201](https://github.com/arviz-devs/arviz/pull/1201))
 * Fix pareto k threshold typo in reloo function ([1580](https://github.com/arviz-devs/arviz/pull/1580))
 * Preserve shape from Stan code in `from_cmdstanpy` ([1579](https://github.com/arviz-devs/arviz/pull/1579))
+* Enabled pylint `use-a-generator` and `consider-using-generator` checks ([1588](https://github.com/arviz-devs/arviz/pull/1588))
```

Comment:
I would write something like `Use generators instead of lists whenever possible`, users reading the changelog and using arviz may not even know what pylint is, not sure they would understand what this change is.

Reply:
Sure, that makes sense.